src/System.Net.Requests/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap-Windows_NT;
```

Comment:
@davidsh we are trying to have the tests compile as netstandard if the test code don't have a code require cross compiling. there is no good reason to add configuration different than netsatndard if there is no code changes between UAP. 

>  I wanted to make sure it used the right binaries.

This shoudl work without adding the UAP configuration to S.Net.Requests as @danmosemsft 

CC @weshaggard